### ISSUE TITLE
ng update don't work with local modules

### ISSUE THREAD

**Issue by lichkovskyi** [angular/angular-cli#12632]
### Bug Report or Feature Request (mark with an `x`)
```
- [x ] bug report -> please search issues before submitting
- [ ] feature request
```

### Command (mark with an `x`)
```
- [ ] new
- [ ] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [x ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Versions

➜  ~ node --version
v10.12.0
➜  ~ npm --version
6.4.1
➜  ~ ng --version

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/

Angular CLI: 6.2.5
Node: 10.12.0
OS: darwin x64
Angular:
...

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.8.5
@angular-devkit/core         0.8.5
@angular-devkit/schematics   0.8.5
@schematics/angular          0.8.5
@schematics/update           0.8.5
rxjs                         6.2.2
typescript                   2.9.2

### Repro steps

1. I added module to package.json from local path (https://docs.npmjs.com/files/package.json#local-paths)
   For example:
          "my-module": "file:../modules/my-module",
2. Run command `ng update`

### The log given by the failure

➜  ~ ng update
Not found : my-module

**Comment by alan-agius4** [comment]
Relates to https://github.com/angular/angular-cli/issues/12263

**Comment by rsarky** [comment]
Hey @alan-agius4 I could give this a shot. Any pointers?

**Comment by alan-agius4** [comment]
@rsarky this should be fixed by https://github.com/angular/angular-cli/pull/12526

**Comment by alan-agius4** [comment]
Fixed by #12526
